File: Exercicios-Beecrowd/C#/1011.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Teste
{
    internal class Program
    {
        static void Main(string[] args)
        {
            double r = Convert.ToDouble(Console.ReadLine());
            double resultado = (4.0 / 3) * 3.14159 * (Math.Pow(r, 3));
            string final = resultado.ToString("F3");
            Console.WriteLine("VOLUME = {0}",final); 

        }
    }
}

File: Exercicios-Beecrowd/Kotlin/1011.kt
import java.util.*

fun main(args: Array<String>) {

	val valor = readLine().toString()
    val teste=valor.toDouble()
    val resultado=(4.0/3)*3.14159*(teste*teste*teste)
    val formatado=("%.3f".format(resultado))
    println("VOLUME = ${formatado}")
}